content/developer/reference/backend/views.rst

```diff
@@ -2527,6 +2527,132 @@ For instance if ``foo`` and ``bar`` refer to two groupbys
 
 has the effect to activate first ``bar`` then ``foo``.
 
+View customizations
+-------------------
+
+As seen in :ref:`Customizing an existing view <reference/js_class>` we can create a custom version
+of a generic view. Here we will present some of these views:
+
+Settings View
```

Comment:
I think I'd place this section inside the Form View section (thus removing the above paragraph)